stdlib/sysconfig.pyi

```diff
@@ -15,11 +16,15 @@ __all__ = [
     "parse_config_h",
 ]
 
+@overload
+@deprecated("SO is deprecated, use EXT_SUFFIX")
+def get_config_var(name: Literal["SO"]) -> Any: ...
+@overload
 def get_config_var(name: str) -> Any: ...
 @overload
-def get_config_vars() -> dict[str, Any]: ...
+def get_config_vars() -> dict[str, Any]: ...  # type: ignore[overload-overlap]
 @overload
-def get_config_vars(arg: str, *args: str) -> list[Any]: ...
+def get_config_vars(*args: str) -> list[Any]: ...
```

Comment:
mypy says overloads overlap, but also gives the right return type

I could instead do
```suggestion
def get_config_vars(arg: str, /, *args: str) -> list[Any]: ...
```